cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -442,30 +444,37 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 				],
 			)),
 			fun: Fungible(WETH_AMOUNT),
-		}];
-		let multi_assets = VersionedAssets::V4(Assets::from(assets));
+		};
+		let assets = vec![fee_asset.clone(), weth_asset.clone()];
 
-		let destination = VersionedLocation::V4(Location::new(
-			2,
-			[GlobalConsensus(Ethereum { chain_id: CHAIN_ID })],
-		));
+		let destination = Location::new(2, [GlobalConsensus(Ethereum { chain_id: CHAIN_ID })]);
 
-		let beneficiary = VersionedLocation::V4(Location::new(
+		let beneficiary = Location::new(
 			0,
 			[AccountKey20 { network: None, key: ETHEREUM_DESTINATION_ADDRESS.into() }],
-		));
+		);
 
+		let xcm_on_bh = Xcm(vec![
+			BuyExecution { fees: fee_asset.clone(), weight_limit: Unlimited },
+			DepositAsset { assets: Wild(AllCounted(2)), beneficiary },
+		]);
+
+		let xcms = VersionedXcm::from(Xcm(vec![
+			WithdrawAsset(assets.clone().into()),
+			SetFeesMode { jit_withdraw: true },
+			InitiateReserveWithdraw {
```

Comment:
OK, this looks good, fairly simple :+1:

Can you check what we would need for Polkadot-native assets? I suspect its `TransferReserveAsset`

Reply:
Exactly.